packages/plugins/src/plugins/morpho/constants.ts

```diff
@@ -18,3 +21,13 @@ export const Underlying = {
 
 export const wethAddress = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
 export const wethDecimals = 18;
+
+export const morphoApiUrl = 'https://blue-api.morpho.org/graphql';
+export const morphoRewardsApiUrl = 'https://rewards.morpho.org/v1';
+
+export const morphoMarketsCachePrefix = `${platformId}-markets`;
+export const morphoVaultsCachePrefix = `${platformId}-vaults`;
+
+export const networkIdToMorphoContract = {
+  [NetworkId.ethereum]: '0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb',
+} as unknown as Record<EvmNetworkIdType, Address>;
```

Comment:
```suggestion
export const networkIdToMorphoContract: Record<EvmNetworkIdType, Address> = {
  [NetworkId.ethereum]: '0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb',
};
```
Explicitly typing the const straight from the get-go would be better, right ?

Reply:
I made a custom type